app-misc/rdfind/rdfind-1.5.0.ebuild

```
@@ -0,0 +1,24 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
```

```suggestion
WANT_AUTOMAKE=1.15
inherit autotools
```
or even
```suggestion
WANT_AUTOMAKE=
inherit autotools
```
is enough. I see upstream requires 1.15 at minimum.

(Note that this is currently broken on EAPI-8, but it's reported and likely fixed before you read this message)

---

Fixed the compile error (I think I accidentally tested the old version before 🤦) and set `WANT_AUTOMAKE`. Although I [couldn't find](https://github.com/pauldreik/rdfind/search?q=1.15) a version requirement upstream?